src/main/java/io/strimzi/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -521,4 +537,41 @@ private void handleCommit(AsyncResult<Void> commitResult) {
             this.commitHandler.handle(commitResult);
         }
     }
+
+    private void handleManualRecords(KafkaConsumerRecords<K, V> record) {
+        if (this.manualRecordBatchHandler != null) {
+            this.manualRecordBatchHandler.handle(record);
+        }
+    }
+
+    private void handleCommitedOffsets(AsyncResult<Map<TopicPartition, io.vertx.kafka.client.consumer.OffsetAndMetadata>> offsetsData) {
```

Comment:
This parameter has to be an `Handler<AsyncResult<Map....>` as the related `commitOffsetsHandler` field.